tests/foreman/api/test_user.py

```diff
@@ -399,6 +399,37 @@ def test_negative_create_with_blank_authorized_by(self):
         with pytest.raises(HTTPError):
             entities.User(auth_source='').create()
 
+    @pytest.mark.tier1
+    def test_positive_table_preferences(self, module_target_sat):
+        """Create a user, create their Table Preferences, read it
+
+        :id: 10fda3f1-4fee-461b-a413-a4d1fa098a94
+
+        :expectedresults: Table Preferences can be accessed
+
+        :CaseImportance: Medium
+
+        :customerscenario: true
+
+        :BZ: 1757394
+        """
+        existing_roles = entities.Role().search()
+        password = gen_string('alpha')
+        user = entities.User(role=existing_roles, password=password).create()
+        name = "hosts"
+        columns = ["power_status", "name", "comment"]
+        sc = ServerConfig(auth=(user.login, password), url=module_target_sat.url, verify=False)
+        entities.TablePreferences(sc, user=user, name=name, columns=columns).create()
+        table_preferences = entities.TablePreferences(sc, user=user).search()
```

Comment:
non blocking: can we do something like this?
```suggestion
        table_preferences = entities.TablePreferences(sc, 
        user=user,name=name,columns=columns).create().search()
```

Reply:
It's perhaps the same, but I like the original logic better: it clearly shows that I first create something, then I search for a list of those things independently.

Reply:
Try to avoid the direct use of entities. We are trying to remove this dependency from robottelo as much as possible.

Reply:
I know and I did it consistently with the rest of the module:
```
$ grep -o entities tests/foreman/api/test_user.py | wc -l
71
```